assets/js/localization.js

```diff
@@ -115,6 +117,8 @@ $(document).ready(function () {
             window.history.replaceState({}, document.title, newURL);
         }
     }
+
+    $('#localeDropdownCaret').css('left', rtlLanguages.indexOf(locale) !== -1 ? '5%' : '90%');
```

Comment:
This is the positioning of the `caret` in the `locale` dropdown button. I am fixing up the position of localeName and the caret.

![screenshot from 2017-07-13 01-01-19](https://user-images.githubusercontent.com/8894636/28136300-d751a7c8-6766-11e7-8569-da2a3a285e36.png)

![screenshot from 2017-07-13 01-01-31](https://user-images.githubusercontent.com/8894636/28136304-d9cc1376-6766-11e7-8562-f57505e5bcb0.png)

It is the position of the carets in the above 2 pics.